NEWS.md

```diff
@@ -8,6 +8,8 @@
 
 - In `chrome_print()`, set a timeout when R tries to detect headless Chrome.
 
+- Added support for Sass files.
```

Comment:
```suggestion
- Added support for Sass files with `.scss` and `.sass` extension (thanks, @felipecrp, #244).
```

We usually thanks contributor in the NEWS item - so thank you ! :)

Reply:
You´re welcome 😄